builder/qemu/step_type_boot_command.go

```diff
@@ -157,11 +167,7 @@ func vncSendString(c *vnc.ClientConn, original string) {
 			log.Printf("Special code '<leftAltOn>' found, replacing with: %d", keyCode)
 
 			c.KeyEvent(keyCode, true)
-			time.Sleep(time.Second / 10)
```

Comment:
it looks like the default delay was 200ms in these cases? Changing it to 100ms doesn't seem to likely to cause problems, but did want to point it out.